src/IO/HDF5/HDF5IOHandler.cpp

```diff
@@ -399,8 +399,46 @@ HDF5IOHandlerImpl::openFile(Writable* writable,
 }
 
 void
-HDF5IOHandlerImpl::openPath(Writable* writable,
-                            Parameter< Operation::OPEN_PATH > const& parameters)
+HDF5IOHandlerImpl::closeFile(
+    Writable * writable,
+    Parameter< Operation::CLOSE_FILE > const & )
+{
+    auto fileID_it = m_fileIDs.find( writable );
+    if( fileID_it == m_fileIDs.end() )
+    {
+        return;
```

Comment:
Should that warn? Did you already see that happen? I fear such things will make other problems hard to debug, so just double-checking.

Reply:
I think we can let that throw instead. Closing a file should require that the file is present in the first place.?